test/runtests.jl

```diff
@@ -2,30 +2,26 @@ using WaterModels
 import InfrastructureModels
 import Memento
 
+# Suppress warnings during testing.
+Memento.setlevel!(Memento.getlogger(InfrastructureModels), "error")
```

Comment:
Looking at this makes me think we should probably add the `logger_config!` function to InfrastructureModels.